tests/acm_observability/test_acm_observability.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from const import HUB_CLUSTER
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+pytestmark = pytest.mark.acm_observability
+
+
+class TestACMObservability:
+    def test_hub_etcd_metrics_exists(
```

Comment:
```suggestion
    def test_hub_etcd_metrics_exist(
```

the current test does not check that the metrics exist